src/dbus/polkit.rs

```diff
@@ -0,0 +1,90 @@
+use std::collections::HashMap;
+use zbus::zvariant::{OwnedValue, Value};
+use zbus::{fdo, message::Header, Connection, Proxy};
+
+pub async fn check_polkit(hdr: Option<Header<'_>>, action_id: &str) -> fdo::Result<()> {
+    let connection = Connection::system()
+        .await
+        .map_err(|e| fdo::Error::Failed(format!("Failed to acquire dbus connection: {}", e)))?;
+    check_polkit_with_connection(hdr, action_id, connection).await
+}
```

Comment:
We'll need an escape hatch to ensure we don't break compatibility with existing clients (e.g. SteamOS, OpenGamepadUI, etc.). I think using an environment variable to enable polkit auth (that defaults to "off") would be a good way for us to pull this feature in and open a path forward for more secure defaults once these clients adopt polkit auth:

```suggestion
pub async fn check_polkit(hdr: Option<Header<'_>>, action_id: &str) -> fdo::Result<()> {
    let polkit_enabled = match std::env::var("ENABLE_POLKIT_AUTH") {
        Ok(value) => value.as_str() == "1",
        Err(_) => false,
    };
    if !polkit_enabled {
        return Ok(());
    }

    let connection = Connection::system()
        .await
        .map_err(|e| fdo::Error::Failed(format!("Failed to acquire dbus connection: {}", e)))?;
    check_polkit_with_connection(hdr, action_id, connection).await
}
```

Reply:
A compile-time flag also works. Thanks!